components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -155,70 +180,314 @@ The complete policy should look like:
 Near the bottom of the page select the **Add Member Expression** button.
 
 Enter a member expression using the format `team:<type>:<name>` or `user:<type>:<name>`; note these are case-sensitive.
 
-* The `<type>` expression is either `ldap` or `saml` according to whichever [external identity provider]({{< relref "configuration.md#authentication-via-existing-identity-management-systems" >}}) Chef Automate has been configured to use.
+* The `<type>` expression is either `ldap` or `saml`.
 * The `<name>` expression is the name of the user or team that the external identity provider knows. For example, this is a valid member expression `team:ldap:editors_team_1`, assuming the `editors_team_1` team is known by your identity provider.
 
 Alternately, you may add *all* teams to a policy using a wildcard as the last term in the member expression: `team:ldap:*` or `team:saml:*`.
 
-The member expression dialog also supports tokens, because candidate tokens are not listed for you either.
+The member expression dialog also supports tokens.
 You enter a token using the expression `token:<id>`.
-In order to find a token's ID, use the command in the API reference for [listing tokens]({{< relref "iam-v2-api-reference#listing-tokens" >}}).
+In order to find a token's ID, visit the *API Tokens* page.
 
-## Removing Legacy Policies
+### Projects
 
-Once you've rewritten your v1 policies as v2 policies, you should remove the v1 legacy policies.
-This section walks you through deleting all of the legacy policies imported into v2, which will provide you a clean slate for exploring the IAM v2 Beta.
+Projects are used to group and permission Chef Automate resources as well as ingested data, specifically Compliance reports, Chef Server and Compliance events, and Infrastructure nodes.
+
+Projects can be managed via the Projects list under the *Settings* tab and consist of an ID, a name, and a collection of ingest rules. Project ingest rules are lists of conditions used only when
+[assigning ingested resources to projects]({{< relref "iam-v2-guide#assigning-ingested-resources-to-projects" >}}),
+so they are not relevant when assigning IAM resources such as teams or roles.
+
+#### Creating a Project
+
+To create a project, navigate to the Projects list under the **Settings** tab and select **Create Project**. You will need to provide a name and can optionally edit the ID. You must create a project before you can assign any resources to it.
+
+#### Assigning Resources to Projects
+
+To assign a team to projects, select a team from the _Teams_ list, then select **Details**.
+Likewise, to assign a token to projects, select a token from the API tokens list, then select **Details**.
+In either case, you can select projects from the projects dropdown to assign.
+
+You may also assign teams and tokens to projects on creation. In the creation modal, select any projects to which the new resource should belong.
```

Comment:
Seems like we are saying you can only assign projects on creation here.

Reply:
tried to give more context above to make it clear its not only on creation